cvmfs/swissknife_sync.cc

```diff
@@ -479,6 +479,7 @@ int swissknife::CommandSync::Main(const swissknife::ArgumentList &args) {
   params.stratum0 = *args.find('w')->second;
   params.manifest_path = *args.find('o')->second;
   params.spooler_definition = *args.find('r')->second;
+  params.is_balanced = *args.find('A')->second == "true";
```

Comment:
This should be a switch (a parameter without an argument).  So if you find the `-A` in the command line, it means that auto balancing is on.